src/main/java/uk/gov/register/db/mappers/ItemMapper.java

```diff
@@ -23,7 +21,7 @@ public ItemMapper() {
     @Override
     public Item map(int index, ResultSet r, StatementContext ctx) throws SQLException {
         try {
-            return new Item(new HashValue(HashingAlgorithm.SHA256, r.getString("sha256hex")), objectMapper.readValue(r.getString("content"), JsonNode.class));
+            return new Item(objectMapper.readValue(r.getString("content"), JsonNode.class));
```

Comment:
I don't think this needs to be changed. I think it's better to read the hash from the database than to recompute it.